scipy/stats/morestats.py

```diff
@@ -1816,6 +1838,13 @@ def rootfunc(ab, xj, N):
         logsf = distributions.gumbel_r.logsf(w)
         sig = array([25, 10, 5, 2.5, 1])
         critical = around(_Avals_gumbel / (1.0 + 0.2/sqrt(N)), 3)
+    elif dist == 'weibull_min':
+        c, loc, scale = distributions.weibull_min.fit(y, loc=0.0)
```

Comment:
why do you use 0 as a guess for the mean?

Reply:
https://github.com/scipy/scipy/issues/11806 because it is the way to make MLE work for 3 param weibull

Reply:
We now use the analytical method of moments solution as a guess in `weibull_min.fit`